external/dear_imgui/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.10)
 project(imgui)
 
-if (APPLE OR LINUX OR UNIX)
+if (APPLE OR LINUX)
```

Comment:
This has to be `if (APPLE OR LINUX OR UNIX)` to work for some reason.

Reply:
This one has to be fixed as well (there are two checks in the same file)

Reply:
Perhaps there should be
```
if (APPLE OR LINUX OR UNIX)
target_include_directories(quaesar PRIVATE ${SDL2_INCLUDE_DIRS})```
Instead of
`target_link_libraries(imgui PRIVATE ${SDL2_LIBRARIES})`